src/Cake.NuGet/Cake.NuGet.csproj

```diff
@@ -1,12 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Description>Provides aliases (extension methods on Cake context) that support CI, build, unit tests, zip, signing, etc. for Cake.</Description>
```

Comment:
Perhaps:
`Cake Module providing NuGet install capabilities for i.e. preprocessor directives like #addin, #tool & #load`
?